pkg/aggregator/aggregator.go

```diff
@@ -354,11 +358,46 @@ func NewBufferedAggregator(s serializer.MetricSerializer, eventPlatformForwarder
 		flushAndSerializeInParallel: NewFlushAndSerializeInParallel(pkgconfigsetup.Datadog()),
 
 		filterListChan: make(chan utilstrings.Matcher),
+		tagFilterList:  loadTagFilterList(),
 	}
 
 	return aggregator
 }
 
+func loadTagFilterList() *filterlist.TagMatcher {
+	tagFilterListInterface := pkgconfigsetup.Datadog().GetStringMap("metric_tag_filterlist")
+
+	tagFilterList := make(map[string]filterlist.MetricTagList, len(tagFilterListInterface))
+	for metricName, tags := range tagFilterListInterface {
+		// Tags can be configured as a simple array of tags - default to exclude.
+		arr, ok := tags.([]string)
+		if ok {
+			tagFilterList[metricName] = filterlist.MetricTagList{
+				Tags:   arr,
+				Action: "exclude",
+			}
```

Comment:
Would it be possible to have one way to configure this? The one below seems more generic and explicit.